scripts/deploy/tests/helm.test.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import child_process from 'child_process';
+import { helmInstallOrUpdate, ensureMedicHelmRepo } from '../src/install.js';
+import UserRuntimeError from '../src/error.js';
+
+const MEDIC_REPO_NAME = 'medic';
+const MEDIC_REPO_URL = 'https://docs.communityhealthtoolkit.org/helm-charts';
+
+describe('helmInstallOrUpdate function', () => {
+  let execSyncStub;
+  let processExitStub;
+  const fakeValuesFile = 'path_to_fake_values.yaml';
+  const fakeNamespace = 'test-namespace';
+  const fakeValues = {
+    project_name: 'test-project',
+    ingress: {
+      host: 'test.host.com'
+    },
+    cht_chart_version: '5.0.0'
+  };
+  const fakeImageTag = 'latest';
+
+  beforeEach(() => {
+    execSyncStub = sinon.stub(child_process, 'execSync');
+    processExitStub = sinon.stub(process, 'exit');
+  });
+
+  afterEach(() => {
+    execSyncStub.restore();
+    processExitStub.restore();
```

Comment:
you can just use `sinon.restore` and it will restore all stubs, you don't need to be explicit.